capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './decorator'
+
+# class capitalize_decorator
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.capitalize
+  end
```

Comment:
Kindly ensure that the CapitalizeDecorator's correct_name method returns the capitalized correct_name. Instead of `@nameable.capitalize`, returning something like this is expected to perform that: `@nameable.correct_name.capitalize`.